src/DBConnection.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * The class creates db connection object using npm client.
+ */
+export default class DBConnection {
+  private static connection;
+
+  /**
+   * Creates and returns db connection object.
+   */
+  public static getConnection(config) {
```

Comment:
Please add parameter and return type

Reply:
Maybe this doesn't need the whole config. Just connection string, user and password should be sufficient.